package.json

```diff
@@ -25,6 +25,7 @@
     "indexer": "turbo index",
     "license": "pnpm zx scripts/license.mjs",
     "lint": "turbo lint",
+    "postlint": "pnpm zx scripts/docblock.mjs",
```

Comment:
Can probably just make this part of the lint command so that turbo can cache it.

Reply:
Does turbo not execute `post*` script automatically?

Reply:
By natural of it running package commands, it might. But I don't think that it will cache the output which is needed to allow it to "skip" processes that don't need to run again.

Reply:
How do we test it to see?

Reply:
I moved it into `lint`.

Reply:
Yea, doing `&& command` still won't cache it in turbo. Need to do something like [Adding tasks](https://turbo.build/repo/docs/crafting-your-repository/configuring-tasks#defining-tasks) and then in the actual `lint` script do something like [running multiple](https://turbo.build/repo/docs/crafting-your-repository/running-tasks#running-multiple-tasks)

Reply:
I added the task to the turbo config - https://github.com/gohypergiant/standard-toolkit/pull/106/files#diff-f8de965273949793edc0fbfe249bb458c0becde39b2e141db087bcbf5d4ad5e3R10-R13.